projects/1.20/assets/doggy-talents-next/doggytalents/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+    "doggytalents.screen.whistler.target.on_duty": "执勤中",
+    "doggytalents.screen.whistler.target.all": "所有狗狗",
+    "doggytalents.screen.whistler.target.title": "哨令目标",
+    "doggytalents.screen.whistler.target.help": "选择哨子生效范围（需先选择模式）",
+    "doggytalents.screen.whistler.mode_help": "模式说明",
+    "doggytalents.screen.conducting_bone.help_title" : "指挥骨菜单帮助",
+    "doggytalents.screen.conducting_bone.no_dog_found" : "本维度无狗狗",
+    "doggytalents.screen.radar.help_title" : "狗狗追踪器帮助",
+    "doggytalents.screen.radar.help" : "请使用鼠标选择要追踪的狗狗。输入狗狗的名字以进行搜索。按下Enter开始追踪。按下Esc取消。",
+    "doggytalents.screen.radar.no_dog_found" : "本维度无狗狗",
+    "doggytalents.screen.heel_by_group.help_title": "组别召唤",
+    "doggytalents.screen.heel_by_group.no_group_found": "未找到组别",
+    "doggytalents.screen.pet_select.exit": "退出互动模式",
+    "doggytalents.screen.pet_select.help.0": "选择互动姿势，非潜行状态再次互动切换",
+    "doggytalents.screen.pet_select.help.1": "潜行+长按右键开始抚摸，第三人称时可用移动键调整视角",
+
+    "amnesia_bone_gui.navbar.general" : "一般的",
+    "amnesia_bone_gui.navbar.talents" : "技能",
+    "amnesia_bone_gui.general.owner_change" : "更换主人",
+    "amnesia_bone_gui.general.untame" : "解除驯服",
+
+    "doggui.common.apply" : "应用",
+    "doggui.common.go_back" : "返回",
+
+    "doggui.navbar.home" : "主页",
+    "doggui.navbar.talents" : "技能",
+    "doggui.navbar.style" : "装扮",
+    "doggui.navbar.stats" : "统计",
+
+    "doggui.home.name" : "名字：",
+    "doggui.home.pet_hint" : "点击抚摸%s！",
+    "doggui.home.edit_info" : "编辑信息",
+    "doggui.home.edit_info.substitude_name" : "称呼：%s",
+    "doggui.home.artifacts" : "神饰",
+    "doggui.home.artifacts.your_artifacts" : "携带的神饰",
+    "doggui.home.artifacts.no_artifacts_in_inv" : "神饰将在此显示",
+
+    "doggui.talents.no_talents_selected" : "请选择初始技能",
+    "doggui.talents.current_talent_level" : "当前等级：",
+    "doggui.talents.cost" : "消耗：",
+    "doggui.talents.insufficent_points" : "技能点不足",
+    "doggui.talents.max_level" : "已满级！",
+    "doggui.talents.train" : "训练",
+    "doggui.talents.invalid.disabled": "该技能已禁用",
+    "doggui.talents.invalid.not_eligible": "不符合学习条件",
+
+    "doggui.style.accessories" : "饰品",
+    "doggui.style.accessories.your_accessories" : "当前饰品",
+    "doggui.style.accessories.no_accessories_in_inv" : "饰品将在此显示",
+    "doggui.style.skins" : "皮肤",
+    "doggui.style.skins.selected" : "已选",
+    "doggui.style.skins.search_by_tags" : "标签搜索",
+    "doggui.style.skins.search_by_name" : "名称搜索",
+    "doggui.style.skins.accessory_state.0" : "这个皮肤还未测试饰品兼容性",
+    "doggui.style.skins.accessory_state.1" : "这个皮肤不兼容任何饰品",
+    "doggui.style.skins.accessory_state.2" : "这个皮肤完美兼容所有饰品",
+    "doggui.style.skins.accessory_state.3" : "这个皮肤仅兼容部分饰品",
+    "doggui.style.skins.accessory_state.4" : "这个皮肤仅兼容模型饰品",
+
+    "doggui.stats.general" : "统计",
+    "doggui.stats.general.damageDealt" : "造成伤害",
+    "doggui.stats.general.distanceInWater" : "水下距离",
+    "doggui.stats.general.distanceOnWater" : "水上距离",
```

```suggestion
    "doggui.stats.general.distanceOnWater" : "水面距离",
```
请与原版描述一致，参见，[统计信息](https://zh.minecraft.wiki/w/%E7%BB%9F%E8%AE%A1%E4%BF%A1%E6%81%AF)。